.github/workflows/_runner-determinator.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       TRIGGERING_ACTOR: ${{ inputs.triggering_actor }}
       ISSUE_OWNER: ${{ inputs.issue_owner }}
       CHECK_EXPERIMENTS: ${{ inputs.check_experiments }}
+      PR_NUMBER: ${{ github.event.pull_request.number }}
```

Comment:
Will this work for trunk-like jobs that are triggered by a tag being added?

Reply:
Yeah, the code queries the labels from PR whenever it run, so the list is always fresh at the time `runnner_determinator.py` runs.  I basically copy this logic from `filter_test_config.py` script that I wrote a while back.  We know that the approach works